src/components/course/CourseRunCard.jsx

```diff
@@ -196,6 +222,17 @@ const CourseRunCard = ({
     courseRun,
   ]);
 
+  const shouldShowLicenseSubsidyPriceText = (
+    !courseRunArchived
+    && !enterpriseConfig.hideCourseOriginalPrice
+    && enrollmentType === enrollButtonTypes.TO_DATASHARING_CONSENT
+    && userSubsidyApplicableToCourse?.subsidyType === LICENSE_SUBSIDY_TYPE
+  );
+  // Only users buckted in `Variation 1` can see the change.
```

Comment:
What is variation 1?

Reply:
Please see the PR description for all the details.